crates/api/src/site/purge/person.rs

```diff
@@ -26,18 +28,17 @@ pub async fn purge_person(
 
   // Read the person to get their images
   let person_id = data.person_id;
-  let person = Person::read(&mut context.pool(), person_id).await?;
 
-  if let Some(banner) = person.banner {
-    purge_image_from_pictrs(&banner, &context).await.ok();
-  }
+  let local_user = LocalUserView::read_person(&mut context.pool(), person_id).await?;
+  let pictrs_uploads =
+    ImageUpload::get_all_by_local_user_id(&mut context.pool(), &local_user.local_user.id).await?;
 
-  if let Some(avatar) = person.avatar {
-    purge_image_from_pictrs(&avatar, &context).await.ok();
+  for upload in pictrs_uploads {
+    delete_image_from_pictrs(&upload.pictrs_alias, &upload.pictrs_delete_token, &context)
+      .await
+      .ok();
   }
```

Comment:
This will potentially be called for hundreds or even thousands of images. You could delete multiple in parallel with join_all, but not sure if pictrs handles that well. Or use a background thread with `spawn_try_task`. cc @phiresky

Reply:
I would limit concurrency here to a reasonable number like 32.

Look into tokio's JoinSet

Reply:
There are some other unavoidable loops in purging, this is pry good for now.